theme/themes/pxt/collections/menu.overrides

```diff
@@ -33,10 +33,6 @@
     background: @invertedBackground;
 }
 
-#mainmenu > .menu > .ui.item:hover > .icon {
-    transform: scale(1.2);
-}
```

Comment:
This change is going to impact more than just the exit debug button. I think we'll need to do something targeted for ".exit-debugmode-btn". Possibly using the "not" css attribute.

Reply:
I did it using the not selector, I don’t know why I didn’t do it right away...